pufferlib/ocean/trash_pickup/trash_pickup.c

```diff
@@ -0,0 +1,74 @@
+#include <time.h>
+#include "trash_pickup.h"
+
+#define INDEX_C(env, x, y) ((y) * (env).grid_size + (x))
+
+// Demo function for visualizing the TrashPickupEnv
+
+
+void demo(int grid_size, int num_agents, int num_trash, int num_bins, int max_steps) {
+    CTrashPickupEnv env;
```

Comment:
Check other env demos for nice initializer syntax

Reply:
Updated (to be pushed).